packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -59,3 +59,21 @@ def test_ip_per_container(cluster):
         app_port = app_definition['container']['docker']['portMappings'][0]['containerPort']
         cmd = '/opt/mesosphere/bin/curl -s -f http://{}:{}/ping'.format(service_points[1].ip, app_port)
         ensure_routable(cmd, service_points)()
+
+
+def test_ip_per_container_with_named_vip(cluster):
+    """Test if we are able to connect to a task with ip-per-container mode
+    """
+    # Launch the test_server in ip-per-container mode
+
+    app_definition, test_uuid = cluster.get_test_app_in_docker(ip_per_container=True)
+
+    assert len(cluster.slaves) >= 2, "IP Per Container tests require 2 private agents to work"
```

Comment:
Any reason we need this? For Load balancing testing, we shouldn't need the 2nd container to actually be on the overlay.